### PR TITLE
Fixing issue with the serialiser

### PR DESCRIPTION
**Pull request by arctouch-caiofonseca**

### CODE REVIEW

File: UaiTomationClient/UaiTomationClient/ArduinoService.m [hunk 1]
```diff
@@ -41,6 +41,8 @@ - (instancetype)init
     self = [super init];
     if (self) {
         _manager = [AFHTTPRequestOperationManager manager];
+        self.manager.responseSerializer = [AFCompoundResponseSerializer serializer];
+        self.manager.responseSerializer.acceptableContentTypes = [NSSet setWithObject:@"text/html"];
```
Comment (arctouch-brunopinheiro) [inline UaiTomationClient/UaiTomationClient/ArduinoService.m]: You could just use the _manager directly here. we are inside the initializer and I think we should either use ```self.manager``` or ```_manager``` for all cases. you just need to choose :)

**Comment by arctouch-brunopinheiro** [comment]
reviewed

File: UaiTomationClient/UaiTomationClient/ArduinoService.m (hunk 1 above)
Comment (arctouch-diegosantiviago) [inline UaiTomationClient/UaiTomationClient/ArduinoService.m]: vai trabalhar manga

Comment (arctouch-caiofonseca) [inline UaiTomationClient/UaiTomationClient/ArduinoService.m]: mala

**Pull request merged**